h/features.py

```diff
@@ -66,6 +72,9 @@ def get_by_name(cls, name):
         """Fetch a flag by name."""
         return cls.query.filter(cls.name == name).first()
 
+    def __repr__(self):
+        return '<Feature %s everyone=%s>' % (self.name, self.everyone)
```

Comment:
I'm curious why you still use `%s` string formatting? It's supposed to be [going away eventually in Python 3](https://docs.python.org/3.2/tutorial/inputoutput.html#old-string-formatting), isn't it?